### PR TITLE
Create .travis.yml

### PR DESCRIPTION
**Pull request by Mariatta**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    # This happenes when other CI (AppVeyor etc.) has already uploaded
+    # the very same dist (usually sdist).
+    skip-existing: true
+    user: &pypi-user core-workflow
+    password: &pypi-password
+      # Encrypt with `travis encrypt -r python/core-workflow --org` while using travis-ci.org;
+      # change to `travis encrypt -r python/core-workflow --api-endpoint 'https://api.travis-ci.com/'`
+      # upon switch to __free__ travis-ci.com:
+      secure: "bg1ZOOHlhaT+S3TQy2WpSsWekHaeZSq39kkZj/Ql3nvJ0ES+a+lTjk+gmuSnSX0Sf9V+88TdN5g1frYVcvYWwIQvLEQP2xhqIfq8K+MVe+GbshQGiTD4PC4f7vRcDY+B93X7cQ8HgkGho3c0/BaEaK6MtrS8WK7dIdHCPaea5D8rLF9jXtS2Lt8MYtqRDG1R2BsNN+o87eyr3qD3aJW6cFRPTHPr/x+gmlKuIJcVOwuVd4xyR701xRpE7hK5/bKvQwI2QZhD7wTyJwYqsl3Ce/ZlyS+IINktkFhD2BqH2ya4tdOHeVhA+b44eJEDXUR7gweyO+Ein5W3S3R2lDxHuX3Aq7nJBKHxU39kwcnPPaMRWYKey51d9zbIicmxjj5sLY0vPYmlX1zdbDUK9NqkdRf8aLzyowWFl3DtkXmyoakITOz9boqDIRiivTJQ/mLG8AFKgf0ms/z1yW4/PN+LoomdpmhQj05EfAJdAvdTw+ceteaeBSKc2uDql4Qj5AKUddeaoLQuPbWForfSPosv9DZZmvOXrmncY91HngrdclxFIssMtefQrC2USbQEIedih1cxfsXzwBrX7+f4eHH2MJt/GrOyNB0gaXc4sikYkaBKsIg7UCWC1/F4jmGkyDLlwp31zflZkjPKUJ3vp2SC9R6sHcT04tvxTVIMDCJgkc8="
```
Comment (webknjaz) [inline .travis.yml]: This should be re-encrypted for with the new repo's private key.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+before_install:
+- &install-flit >-
+  pip install --upgrade pip flit
+
+.mixtures:
+- &run-if-tagged
+  if: tag IS present
+- &run-if-cherry-picker
+  if: tag =~ ^cherry\-picker\-v\d+\.\d+\.\d+$
```
Comment (webknjaz) [inline .travis.yml]: This can be hugely simplified now

File: .travis.yml (hunk 1 above)
Comment (Mariatta) [reply .travis.yml]: I don't remember how this got generated. @brettcannon I think you did the encryption last time?

Comment (brettcannon) [reply .travis.yml]: I don't remember doing it. 😁 I never set up any of the deployment stuff.

Comment (webknjaz) [reply .travis.yml]: You'd need to use that command in the comment.

Comment (webknjaz) [reply .travis.yml]: @Mariatta generated it previous time: https://github.com/python/core-workflow/pull/262#commits-pushed-fddf8dc

Comment (webknjaz) [reply .travis.yml]: (https://github.com/python/core-workflow/pull/262/commits/fddf8dcc495c17356eb8855af5b4766257d2e76d)

Comment (Mariatta) [reply .travis.yml]: 😅 totally forgot about this. I'll look into it later today.

File: .travis.yml (hunk 2 above)
Comment (Mariatta) [reply .travis.yml]: Can you suggest how? I'm not actually very well-versed in config file like this 😅

Comment (webknjaz) [reply .travis.yml]: Just use that other anchor without a regex

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,87 @@
+conditions: v1
+
+git:
+  depth: false
+
+language: python
+
+dist: trusty
```
Comment (webknjaz) [inline .travis.yml]: Let's not test in the old VM
```suggestion
dist: xenial
```

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+  python: "3.7"
+- &install-and-test-cherry-picker
+  <<: *run-if-cherry-picker-or-untagged
+  env:
+    TARGET_PKG: cherry_picker
+  install:
+  - flit install
+  script:
+  - pytest cherry_picker/test.py -v
+  - popd
```
Comment (webknjaz) [inline .travis.yml]:
```suggestion
```

**Comment by webknjaz** [comment]
Now that there's no blurb, script+install could be moved to the root level and most of the anchors aren't needed so the config could be ~two times smaller.

**Comment by webknjaz** [comment]
If you want, I can take care of simplifying this config.

**Comment by Mariatta** [comment]
> If you want, I can take care of simplifying this config.

Yes! I'll appreciate this. Thanks!

**Comment by webknjaz** [comment]
#11 should do

**Pull request closed without merging**